test/dlc_tests/ecs/pytorch/inference/test_ecs_pytorch_inference.py

```diff
@@ -36,6 +36,8 @@ def test_ecs_pytorch_inference_eia(pytorch_inference_eia, ecs_container_instance
     public_ip_address = ec2_utils.get_public_ip(worker_instance_id, region=region)
 
     model_name = "pytorch-densenet"
+    if "1.3.1" in pytorch_inference_eia:
```

Comment:
Use https://github.com/aws/deep-learning-containers/blob/048d28b15d6b87796067428296668b5a8b8d5738/test/test_utils/__init__.py#L655-L676 to find the framework version of the image, rather than searching for the exact string in the `image_tag`.

This will allow easier unit testing, and will also allow us to improve the implementation that finds the framework version on an image URI everywhere in the code, if needed.